hazelcast-client/src/main/java/com/hazelcast/client/spi/impl/ClientInvocationServiceSupport.java

```diff
@@ -43,16 +48,24 @@
 import java.util.concurrent.BlockingQueue;
 import java.util.concurrent.ConcurrentHashMap;
 import java.util.concurrent.ConcurrentMap;
-import java.util.concurrent.LinkedBlockingQueue;
 import java.util.concurrent.TimeUnit;
 
 import static com.hazelcast.client.spi.properties.ClientProperty.MAX_CONCURRENT_INVOCATIONS;
 import static com.hazelcast.instance.OutOfMemoryErrorDispatcher.onOutOfMemory;
 import static com.hazelcast.util.StringUtil.timeToString;
 import static java.lang.String.format;
+import static java.util.concurrent.TimeUnit.MICROSECONDS;
+import static java.util.concurrent.TimeUnit.NANOSECONDS;
 
 
 abstract class ClientInvocationServiceSupport implements ClientInvocationService {
+    public static final HazelcastProperty FAST
```

Comment:
I would like to do some change. Have a string value:

1: blocking
2: temp spinning
3: spin

This gives us some flexibility for playing with low latency settings. By default it is blocking; but from performance experiments I have seen that temp spinning can lead to a significant performance improvement with throughput tests.